.github/workflow_scripts/env_setup.sh

```diff
@@ -9,3 +9,15 @@ function install_rag_test {
     python3 -m pip install --upgrade -e "./[tests]"
     python3 -m pip install pytest
 }
+
+function setup_build_contrib_env {
+    python3 -m pip install --upgrade pip
+    python3 -m pip install -r $(dirname "$0")/../../docs/requirements_doc.txt
+    python3 -m pip install git+https://github.com/zhanghang1989/d2l-book
```

Comment:
Is this still in use?

Reply:
I have removed `git+https://github.com/zhanghang1989/d2l-book`